.github/workflows/automationTemplate.yaml

```diff
@@ -53,28 +53,18 @@ jobs:
       - name: Run SDK Automation
         run: dotnet run --project build/tools/Automation/Automation.csproj ${RELEASE_DATE} ${{ github.workspace }}
 
-      - name: Check if any git changes exist.
-        run: |
-          changes=$(git diff-index --name-only HEAD --)
-          echo ${changes}
-          if [ -z "${changes}" ]; then
-              echo "No git changes to commit. Cancelling workflow..."
-              exit 1
-          fi
-          git status
-
       - name: Commit new ${{ env.PLATFORM_NAME_CAMEL_CASE }} SDK versions
         run: |
-          git status
+          set -x
           git config user.name 'Oryx Team'
           git config user.email 'oryx@microsoft.com'
+          
           git add -A
-          git commit -m "Add new SDK versions"
+          git commit -m "Add new SDK versions" | true
+          gitCommitExitCode=${PIPESTATUS[0]}
+          [[ $gitCommitExitCode = 0 ]] && cancelWorkflow=false || cancelWorkflow=true
```

Comment:
checking the `git commit` exit status is replacing the following check: https://github.com/microsoft/Oryx/pull/1786/files#diff-3ed663dcfb84882e18b67fce2ab60dcb67f0b57ac69932ca286dcea41b5d80a5L56-L65